problems/arrayMethods.js

```diff
@@ -91,15 +149,29 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+  let obj = {}
+  items.forEach((item) => {
+    if (obj[item] = obj[item]? obj[item] + 1 : 1){
```

Comment:
This is kind of confusing because you have a ternary inside of an if statement.

You only want to use one of those